src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs

```diff
@@ -567,11 +567,12 @@ private BinaryLogRecordKind Write(CriticalBuildMessageEventArgs e)
 
         private BinaryLogRecordKind Write(PropertyReassignmentEventArgs e)
         {
-            WriteMessageFields(e, writeMessage: false, writeImportance: true);
+            WriteMessageFields(e, writeMessage: true, writeImportance: true);
```

Comment:
why is this changing to true?

Reply:
previously, the message was provided by logviewer resources.
In this PR I attempted to pass a localized Message from MSBuild